packages/desktop-client/src/components/schedules/index.js

```diff
@@ -70,7 +74,15 @@ export default function Schedules() {
         />
       </View>
 
-      <View style={{ alignItems: 'flex-end', margin: '20px 0', flexShrink: 0 }}>
+      <View
+        style={{
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          margin: '20px 0',
+          flexShrink: 0
+        }}
+      >
+        <Button onClick={onDiscover}>Find schedules…</Button>
```

Comment:
Maybe it's just me.. but this feels.. a bit passive-agressive?..

Hope you get my point.. :) 

```suggestion
        <Button onClick={onDiscover}>Find schedules</Button>
```

Reply:
Fair. I was going for matching the macOS standard of having “…” when the button opens another window rather than performing an action, but the app doesn’t do it anywhere else so it would be inconsistent.